core/src/main/kotlin/com/alecstrong/sql/psi/core/sql.bnf

```diff
@@ -232,9 +232,10 @@ expr ::= ( other_expr
 other_expr ::= extension_expr
 extension_expr ::= NULL
 extension_stmt ::= NULL
-multi_column_expr ::= values_expression ( LT | LTE | GT | GTE ) values_expression {
+multi_column_expr ::= multi_column_expression ( LT | LTE | GT | GTE ) multi_column_expression {
   pin = 2
 }
+multi_column_expression ::= LP expr ( COMMA expr ) + RP
```

Comment:
Multi column expr requires multiple expr `+`, sample: `(1,1) <= (2,2)`. Otherwise, we use `parenExpr` as fallback, because there is no pin.
We can't reuse `values_expression` because you can insert a single value: `INSERT .. VALUES (1)`